sdk/src/main/scala/io/horizen/account/api/rpc/types/TxPoolContent.scala

```diff
@@ -0,0 +1,14 @@
+package io.horizen.account.api.rpc.types
+
+import com.fasterxml.jackson.annotation.JsonView
+import io.horizen.evm.Address
+import io.horizen.json.Views
+
+import java.math.BigInteger
+import scala.collection.concurrent.TrieMap
+import scala.collection.mutable
+
+@JsonView(Array(classOf[Views.Default]))
+class TxPoolContent(
+    val pending: TrieMap[Address, mutable.SortedMap[BigInteger, TxPoolTransaction]],
```

Comment:
In the doc https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-txpool I see that nonce is serialized as decimal. In our case we serialize it as hex (BigInteger is always represented as hex)